pkgs/by-name/az/azure-pipelines-agent/package.nix

```
@@ -0,0 +1,262 @@
+{
+  bash,
+  buildDotnetModule,
+  coreutils,
+  dotnetCorePackages,
+  fetchFromGitHub,
+  gitMinimal,
+  glibc,
+  glibcLocales,
+  icu,
+  krb5,
+  lib,
+  nodejs_20,
+  openssl,
+  stdenv,
+  which,
+  zlib,
+}:
+buildDotnetModule rec {
+  pname = "azure-pipelines-agent";
+  version = "4.259.0";
+
+  src = fetchFromGitHub {
+    owner = "microsoft";
+    repo = "azure-pipelines-agent";
+    rev = "v${version}";
+    sha256 = "sha256-7wd7fYI/cCfAz3usUzH6pyJ5mVAPpo6LNlbXOYcZ47k=";
```

```suggestion
    tag = "v${version}";
    hash = "sha256-7wd7fYI/cCfAz3usUzH6pyJ5mVAPpo6LNlbXOYcZ47k=";
```
fetchers now use these attrs